cray_product_catalog/catalog_update.py

```diff
@@ -232,6 +246,15 @@ def update_config_map(data, name, namespace):
             else:
                 LOGGER.exception("Error calling replace_namespaced_config_map")
 
+def split_catalog_data(data, main_cm_data, sub_cm_data):
+    LOGGER.debug("Splitting cray-product-catalog data")
+    for k in set(list(data.keys())) - set(SUB_CM_FIELDS_LIST):
```

Comment:
this logic has a caveat, assume in main data we are getting few common keys, which has be present in both cm.
In that case those keys will be missing in product config map.